igvc_rviz_plugins/plugin_description.xml

```diff
@@ -1,4 +1,9 @@
 <library path="lib/libigvc_rviz_plugins">
+	<class name="igvc_rviz_plugins/WaypointPanel"
```

Comment:
can you format these additions like the line below. If I had to guess these are tabs